example(datacamp):Add data camp connector example (#242)
fivetran/fivetran_connector_sdk: README.md

- [commonpaper](https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/commonpaper) - This example shows how to sync agreement data from Common Paper using Connector SDK. You need to provide your Common Paper credentials for this example to work.
- [couchbase_capella](https://github.com/fivetran/fivetran_connector_sdk/blob/main/examples/source_examples/couchbase_capella) - This example shows how to sync data from Couchbase Capella using Connector SDK. You need to provide your Couchbase Capella credentials for this example to work.
- [couchbase_magma](https://github.com/fivetran/fivetran_connector_sdk/blob/main/examples/source_examples/couchbase_magma) - This example shows how to sync data from a self-managed Couchbase Server using the Magma storage engine with Connector SDK. You need to provide your Couchbase credentials for this example to work.

> - [datacamp](https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/datacamp) - This is an example to show how to sync course catalog data from DataCamp's LMS Catalog API by using Connector SDK. You need to provide your DataCamp Bearer token for this example to work.

Reviewer: ```suggestion
- [datacamp](https://github.com/fivetran/fivetran_connector_sdk/tree/main/examples/source_examples/datacamp) - This example shows how to sync course catalog data from DataCamp's LMS Catalog API by using the Connector SDK. You need to provide your DataCamp bearer token for this example to work.
```
